examples/image-to-text/run_pipeline.py

```diff
@@ -86,6 +86,12 @@ def main():
     parser.add_argument("--batch_size", type=int, default=1, help="Input batch size.")
     parser.add_argument("--warmup", type=int, default=3, help="Number of warmup iterations for benchmarking.")
     parser.add_argument("--n_iterations", type=int, default=5, help="Number of inference iterations for benchmarking.")
+    parser.add_argument(
+        "--ignore_eos",
+        default=True,
```

Comment:
I would prefer the default is False for backward compatibility

Reply:
It was designed to do the opposite but I'm fine with either way. I just pushed a new commit so that it behaves as you described.